File: java/lib/src/test/java/org/xmtp/proto/message/contents/ProtoTest.java
package org.xmtp.proto.message.contents;

import org.junit.Test;
import org.xmtp.proto.message.contents.Content.Compression;

import static org.junit.Assert.*;

public class ProtoTest {
    @Test public void versionMethodReturnsTrue() {
        assertEqual(Compression.COMPRESSION_DEFLATE_VALUE, 0);
    }
}

File: kotlin/lib/src/test/kotlin/org/xmtp/proto/message/contents/protoTest.kt
package org.xmtp.proto.message.contents

import kotlin.test.Test
import kotlin.test.assertEquals
import org.xmtp.proto.message.contents.*

class ProtoTest {
    @Test fun testTypesAreAvailable() {
        val message = MessageV2Kt()
        assertEquals(false, message.hasV2())
    }
}